connexion/apps/flask_app.py

```diff
@@ -150,24 +152,30 @@ def run(self,
             raise Exception(f'Server {self.server} not recognized')
 
 
-class FlaskJSONEncoder(json.JSONEncoder):
-    def default(self, o):
+class FlaskJSONProvider(DefaultJSONProvider):
+    @classmethod
+    def default(cls, o):
         if isinstance(o, datetime.datetime):
             if o.tzinfo:
                 # eg: '2015-09-25T23:14:42.588601+00:00'
-                return o.isoformat('T')
+                return o.isoformat("T")
             else:
                 # No timezone present - assume UTC.
                 # eg: '2015-09-25T23:14:42.588601Z'
-                return o.isoformat('T') + 'Z'
+                return o.isoformat("T") + "Z"
 
         if isinstance(o, datetime.date):
             return o.isoformat()
 
         if isinstance(o, Decimal):
             return float(o)
 
-        return json.JSONEncoder.default(self, o)
+        return super().default(o)
+
+
+class FlaskJSONEncoder(json.JSONEncoder):
+    def default(self, o):
```

Comment:
Would you please add a comment explaining why this tiny class must exist and override the `default` method?